packages/lib/constants.ts

```diff
@@ -48,6 +48,11 @@ export const GOOGLE_SHEETS_CLIENT_ID = env.GOOGLE_SHEETS_CLIENT_ID;
 export const GOOGLE_SHEETS_CLIENT_SECRET = env.GOOGLE_SHEETS_CLIENT_SECRET;
 export const GOOGLE_SHEETS_REDIRECT_URL = env.GOOGLE_SHEETS_REDIRECT_URL;
 
+export const NOTION_OAUTH_CLIENT_ID = env.NOTION_OAUTH_CLIENT_ID;
+export const NOTION_OAUTH_CLIENT_SECRET = env.NOTION_OAUTH_CLIENT_SECRET;
+export const NOTION_AUTH_URL = env.NOTION_AUTH_URL;
+export const NOTION_REDIRECT_URI = env.NOTION_REDIRECT_URI;
```

Comment:
isn't the NOTION_REDIRECT_URI always the same and `${WEBAPP_URL}/api/...callback`?
If so, please use this instead of a new environment variable